assembly_uploader/ena_queries.py

```diff
@@ -123,12 +123,21 @@ def _get_private_study(self):
         response = self.retry_or_handle_request_error(self.get_request, url)
         study = self.get_data_or_handle_error(response)
         study_data = study["report"]
-        reformatted_data = {
-            "study_accession": study_data["secondaryId"],
-            "study_title": study_data["title"],
-            #   remove time and keep date
-            "first_public": study_data["firstPublic"].split("T")[0],
-        }
+        try:
+            reformatted_data = {
+                "study_accession": study_data["secondaryId"],
+                "study_title": study_data["title"],
+                # remove time and keep date
+                "first_public": study_data["firstPublic"].split("T")[0],
+            }
+        except AttributeError:
+            # if the release date was postponed, firstPublic == None
+            # fallback on holDate
```

Comment:
there is a tiny typo in this line 
```suggestion
            # fallback on holdDate
```